osquery/core/testing.h

```diff
@@ -34,8 +34,9 @@ extern const char *kOsqueryTestModuleName;
 
 /// These are the expected arguments for our test worker process.
 extern const char *kExpectedWorkerArgs[];
+extern const size_t kExpectedWorkerArgsCount;
 
 /// These are the expected arguments for our test extensions process.
 extern const char *kExpectedExtensionArgs[];
+extern const size_t kExpectedExtensionArgsCount;
 }
-
```

Comment:
Is losing this newline gunna mess with things? I couldn't recall, but I thought we needed the trailing newline for some reason.